{{cookiecutter.project_shortname}}/INSTALL.rst

```diff
@@ -1,138 +1,52 @@
-
-The configuration file is located in:
-
-- ``<sys.prefix>/var/instance/invenio.cfg``
-
-In the examples below we assume the remote services are running on the IP
-address ``192.168.33.3``.
+Quick start
+-----------
 
 .. note::
 
-    If you have everything running ``localhost`` without passwords, you can skip
-    the configuration as everything is pre-configured for ``localhost``.
-
-Invenio
-~~~~~~~
-The most important Invenio related configuration to set is:
-
-.. code-block:: python
-
-    SECRET_KEY = '...your-very-secret-key...'
-
-The secret key is used for protecting sessions and encrypting/decrypting
-tokens. For production systems is critical to set this configuration to keep
-your system secure.
-
-Database
-~~~~~~~~
-Next, you need to tell  Invenio which database you want to use. By default for
-development purposes, Invenio will create a SQLite database next to
-``invenio.cfg``. To configure Invenio to use e.g. PostgreSQL set the following
-configuration:
-
-.. code-block:: python
-
-    SQLALCHEMY_DATABASE_URI='postgresql://invenio:invenio@192.168.33.3/invenio'
-
-Search engine
-~~~~~~~~~~~~~
-Next, point Invenio to your Elasticsearch cluster:
-
-.. code-block:: python
+    The docker configuration provided in this module uses PostgresSQL and ElasticSearch 5. If you have chosen different
+    database or ElasticSearch version and you want to try this quick start with Docker, modify the ``setup.py``
+    configuration.
 
-    SEARCH_ELASTIC_HOSTS=['192.168.33.3:9200']
+Install `Docker` and ``docker-compose`` in your machine.
+Then, create and run all the docker containers:
 
-Message broker
-~~~~~~~~~~~~~~
-Next, configure where Invenio can find the RabbitMQ message broker used for
-background jobs and bulk indexing:
-
-.. code-block:: python
-
-    CELERY_BROKER_URL='amqp://invenio:invenio@192.168.33.3:5672/invenio'
-
-
-Cache
-~~~~~
-Last, you need to configure where Invenio can find the Redis cache. The cache
-needs to be set in a couple of variables, as it is being used for different
-purposes (e.g. session store, background job results, caching):
-
-.. code-block:: python
-
-    CACHE_TYPE='redis'
-    CACHE_REDIS_URL='redis://192.168.33.3:6379/0'
-    ACCOUNTS_SESSION_REDIS_URL='redis://192.168.33.3:6379/1'
-    CELERY_RESULT_BACKEND='redis://192.168.33.3:6379/2'
+.. code-block:: console
 
-Note, that each variable above points to a different database number in Redis
-(the ``0``, ``1``, ``2`` suffix in the URL)
+    $ cd {{ cookiecutter.project_shortname }}
+    $ docker-compose -f docker-compose.full.yml up
 
 Database and search index
 -------------------------
 The last you need to do is to create the database tables and search indexes.
-
-Create the database (not needed if the database was created outside of invenio
-already):
+Connect to the web container:
 
 .. code-block:: console
 
-   (ils)$ invenio db init
+    $ docker exec -it {{ cookiecutter.project_shortname }}_web-ui_1 /bin/bash
```

Comment:
Wouldn't something like ``docker-compose run web-ui /bin/bash`` work?

Reply:
Fixed, but I need the name of the project `project-web-ui`